domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs

```diff
@@ -2,6 +2,7 @@
 using System.ComponentModel.DataAnnotations;
 using System.Text;
 using Microsoft.Extensions.DependencyInjection;
+using ProjectOrigin.HierarchicalDeterministicKeys.Implementations;
```

Comment:
This can be deleted?

Reply:
I don't think so. I upgraded project origin, and then it wanted this dependency